src/main/kotlin/no/nav/yrkesskade/ysmeldingapi/client/altinn/AltinnClient.kt

```diff
@@ -45,7 +45,10 @@ class AltinnClient(
 
         if (response.status == Response.Status.OK.statusCode) {
             val altinnReporteeResponse = response.readEntity(AltinnReporteeResponse::class.java)
-            return altinnReporteeResponse.embedded.reportees.filterNot { it.type == "Person" }.map { AltinnOrganisasjonDto.fraAltinnReportee(it) }
+            return altinnReporteeResponse.embedded.reportees
+                .filter { it.status == "Active" }
+                .filterNot { (it.type == "Person")}
```

Comment:
Håper dette fortsatt er gyldig Kotlin?
```suggestion
                .filterNot { it.type == "Person" }
```